lib/veritrans.rb

```diff
@@ -183,13 +177,7 @@ def file_logger
     if !@file_logger
       require 'logger'
       begin
-        if defined?(Rails) && Rails.root
-          require 'fileutils'
-          FileUtils.mkdir_p(Rails.root.join("log"))
-          @file_logger = Logger.new(Rails.root.join("log/veritrans.log").to_s)
-        else
           @file_logger = Logger.new("/dev/null")
-        end
```

Comment:
I was not asking you to remove it actually. I was asking you to test if the behavior will be as expected or not on rails, will explain via slack